packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -244,6 +250,7 @@ export default {
   search,
   send,
   settings,
+  'share-recording': shareRecording,
```

Comment:
`share-recording` is too specific for this icon, i would suggest naming it more generically like `share` or `share-alt` as font awesome does